module/utility/dice.mjs

```diff
@@ -264,6 +265,14 @@ export class HeroRoller {
         return this;
     }
 
+    makeTransformRoll(apply = true) {
```

Comment:
what is your thinking for adding a new roll type that I'm not sure exists in the game?

Reply:
This was built largely to retain symmetry with the Adjustment implementation, which was the main model I based the Transform implementation on. Though I'm not sure what you mean that it doesn't exist in the game (unless you mean it doesn't have a unique procedure in the game, and just uses BaseTerms and BaseTotal)